### ISSUE TITLE
Add test: lib/transform/config.js

### ISSUE THREAD

**Issue by dbanksdesign** [amzn/style-dictionary#143]
Note: Work off of the 'jest' branch. That is our new branch to work off of for the testing upgrades. When everything is ready we will merge that branch back into develop and mainline.

**Comment by didoo** [comment]
@dbanksdesign do you think I can pick up this one after #233 (now I know a little bit more the code in that file) or do you prefer to leave is for first-issues? (personally, I would prefer to see this fully covered, it's an important part of SD).

**Comment by dbanksdesign** [comment]
Please do!

**Comment by chazzmoney** [comment]
Is this now covered? Do we need to flesh out more here?

https://github.com/amzn/style-dictionary/blob/master/__tests__/transform/config.test.js

**Comment by dbanksdesign** [comment]
I think we are covered now. There can be some more test we can put in later, but I think this is a good start

**Comment by chazzmoney** [comment]
Sweet. If there are other tests you want to see added, can you open another issue and I'll add them in?

Thanks!
